caesars-palace/blocks/slider-card/slider-card.css

```diff
@@ -120,12 +133,60 @@
     position: relative;
     padding: 16px;
     display: block;
-    height: 190px;
   }
 
   .slider-card .card .close-button {
-    margin: -27px 20px 20px;
+    margin: 0 20px 20px;
     position: relative;
     z-index: 2;
   }
+
+
+  .slider-card-wrapper .chevron-left {
+    display: flex;
+    align-items: center;
+    position: absolute;
+    z-index: 100;
+    top: 30%;
+    left: -.5%;
+    background-color: white;
+    width: 30px;
+    height: 30px;
+    border-radius: 80%;
```

Comment:
Looks like we still need to add the `box-shadow` for these chevron buttons.

`box-shadow: 0 10px 15px -3px rgba(0,0,0,0.1), 0 4px 6px -2px rgba(0 0 0 / 6%);`